images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -101,6 +101,12 @@ Describe "GoogleCloudSDK" -Skip:(Test-IsWin22) {
     }
 }
 
+Describe "ServiceFabricSDK" -Skip:(Test-IsWin22) {
```

Comment:
I believe we don't need another describe since we already have it
![image](https://user-images.githubusercontent.com/48208649/142860473-f5cd62a9-8c1d-47c3-aeb0-40daa8fbe5a1.png)